src/bootstrap/builder.rs

```diff
@@ -1162,8 +1163,8 @@ impl<'a> Builder<'a> {
             // fixed via better support from Cargo.
             cargo.env("RUSTC_LINT_FLAGS", lint_flags.join(" "));
 
-            rustdocflags.arg("-Dinvalid_codeblock_attributes");
         }
+        rustdocflags.arg("-Winvalid_codeblock_attributes");
```

Comment:
Why was this moved outside of the InTree block? We should still only specify this for in-tree crates.

Reply:
Woups, my bad.